Tools/ChangeLog

```diff
@@ -1,3 +1,16 @@
+2022-04-07  Jonathan Bedard  <jbedard@apple.com>
+
+        [Merge-Queue] Canonicalize parent commits
+        https://bugs.webkit.org/show_bug.cgi?id=238951
+        <rdar://problem/91436607>
+
+        Reviewed by NOBODY (OOPS!).
+
+        * CISupport/ews-build/steps.py:
+        (Canonicalize.run): Canonicalize first 2 parent commits if not rebasing.
```

Comment:
If a commit is already canonicalized, is it safe to canonicalize it again?

Reply:
Yes! canonicalizing an already canonicalized commit will be a no-op, because it will result in the same commit message.

Reply:
(as a side note, on the off chance that's wrong, the consequence in this workflow is not getting the purple merged tag, which isn't a huge deal, we close and comment on the PR anyways)

Reply:
ok